monitor/getBalances.js

```diff
@@ -66,7 +66,13 @@ async function main(bridgeMode, eventsInfo) {
     const mintedCoinsBN = new BN(mintedCoins)
     const burntCoinsBN = new BN(burntCoins)
     const totalSupplyBN = mintedCoinsBN.minus(burntCoinsBN)
-    const foreignErc20BalanceBN = new BN(foreignErc20Balance).plus(lateForeignConfirmationsTotalValue)
+    let foreignErc20BalanceBN = new BN(foreignErc20Balance).plus(lateForeignConfirmationsTotalValue)
+    try {
+      const invested = await foreignBridge.methods.investedAmount(erc20Address).call()
```

Comment:
Does it make sense to set `foreignDelayedBlockNumber` to get the value at a specific moment of time?

Reply:
Yes, it makes total sense. Thanks.